geospaas_processing/copiers.py

```diff
@@ -2,7 +2,7 @@
 import shutil
 import logging
 import time
-
+from os.path import exists
```

Comment:
You use that only twice. Please use it as `os.path.exists`
```suggestion
```

Reply:
@akorosov it was discussed here: https://github.com/nansencenter/django-geo-spaas-processing/pull/35#discussion_r562654203

We can't do that!